lib/dialects/mysql.ts

```diff
@@ -0,0 +1,123 @@
+import Knex from 'knex';
+import { SchemaInspector } from '../types/schema-inspector';
+import { MySQLTable } from '../types/table';
+import { MySQLColumn } from '../types/column';
+
+export default class MySQL implements SchemaInspector {
+  knex: Knex;
+
+  constructor(knex: Knex) {
+    this.knex = knex;
+  }
+
+  async tables() {
+    type RawTable = {
+      TABLE_NAME: string;
+      TABLE_SCHEMA: string;
+      TABLE_COMMENT: string | null;
```

Comment:
why not just make it optional?

Reply:
Optional is `undefined` or key non existent in the response, while we know for a fact that MySQL will return this key with either a `string` or `null` value